setup.py

```diff
@@ -47,12 +47,15 @@ def get_readme() -> str:
     install_requires=[
         "awscli",
         "cloudpickle>=0.5.5",
-        "gym[classic_control]",
+        "gym[atari,classic_control]",
```

Comment:
Why is Atari a requirement?

Reply:
At one point when I was roaming around dependency hell, installing atari helped. But it's no longer necessary to pass the bc and dagger tests -- good catch! -- so I just removed it.